src/components/avatar/Avatar.tsx

```diff
@@ -0,0 +1,22 @@
+import avatar from "../../assets/image-avatar.jpg";
+import Box from "@mui/material/Box";
+import "./Avatar.scss";
+
+export const Avatar = () => {
+  return (
+    <Box
+      sx={{
+        borderTop: "1px solid white",
+        position: "absolute",
+        bottom: "0px",
```

Comment:
if you put 0 then you do not need px. Please try to remember about this :)

Reply:
fixed.